rebar.config

```diff
@@ -1,8 +1,6 @@
 {require_otp_vsn, "R1[456]|^[0-9]+"}.
 
-{deps, [
-    {jsonx, ".*", {git, "https://github.com/iskra/jsonx.git", {branch, master}}}
-]}.
+{deps, [{jsonx, "", {git, "https://github.com/alertlogic/jsonx.git", {branch, master}}}]}.
```

Comment:
using the alertlogic (the company I work for) fork of the abandoned `jsonx` library for rebar3 compatibility without an override. For what it's worth, I'll be deprecating the encode/decode functionality in the library soon and removing in in an upcoming v2 release. I don't know of a library that supports all the formats I want to support, so it makes more sense to simply make some recommendations in the README.